graphql/me/mutation.ts

```diff
@@ -196,11 +184,7 @@ export class MutateMeResolver {
     @Mutation((returns) => Student)
     @Authorized(Role.UNAUTHENTICATED, Role.ADMIN)
     @RateLimit('RegisterStudent', 10 /* requests per */, 5 * 60 * 60 * 1000 /* 5 hours */)
-    async meRegisterStudent(
-        @Ctx() context: GraphQLContext,
-        @Arg('data') data: RegisterStudentInput,
-        @Arg('noEmail', { nullable: true }) noEmail: boolean = false
-    ) {
+    async meRegisterStudent(@Ctx() context: GraphQLContext, @Arg('data') data: RegisterStudentInput, @Arg('noEmail', { nullable: true }) noEmail = false) {
```

Comment:
The :boolean is needed here for TypeGraphQLs reflection (unlile Typescript itself which infers the type from the default parameter) ...

Reply:
Then let's suppress this one as well 🙂